lib/utils/get-proxy-agent.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+const getProxyForUrl = require('proxy-from-env').getProxyForUrl;
```

Comment:
can this use destructuring? 
```suggestion
const {getProxyForUrl} = require('proxy-from-env');
```